adapter/src/ethereum/error.rs

```diff
@@ -1,5 +1,6 @@
 use primitives::adapter::{AdapterErrorKind, Error as AdapterError};
-use primitives::ChannelId;
+use primitives::address::Error as AddressError;
```

Comment:
can you group them together from now on?
E.g. `primitives::{ all imports here }`;